pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1168,6 +1169,8 @@ func Run(ps *pubsub.PubSub, loggerArg *logrus.Logger, logArg *base.LogObject) in
 	// start remote attestation task
 	attestModuleStart(&zedagentCtx)
 
+	zedagentCtx.inMainLoop = true
+
```

Comment:
This seems early. I think we should wait at least until we have processed the initial config from the controller (basically the first inhaleConfig() being done. And potentially longer; don't know if we can easily tell that all of the app instances in the initial config have reached the Booting or Running state, while ignoring any app instances which get deployed after the initial boot.

Reply:
I can see existing comment here https://github.com/lf-edge/eve/blob/50b4b878f212e4ab974eb276e32c5ae543a3f322/api/go/info/info.pb.go#L553-L556 which indicates, that we are not check something from app instances. 
But delay it after inhaleDeviceConfig looks reasonable, done.

Reply:
Actually @petr-zededa the current PR code sets it before calling inhale.
But let's go with this and see if we want to add some more feedback mechanisms between the different microservices later.
For instance, if we could wait until every object with a *Config also has the corresponding Status published it means that all of the initial config has propagated to all the microservices. But that's for later.